solr/prometheus-exporter/src/java/org/apache/solr/prometheus/exporter/SolrExporter.java

```diff
@@ -349,4 +361,31 @@ private static MetricsConfiguration loadMetricsConfiguration(String configPath)
   private static String getSystemVariable(String name) {
     return System.getProperty(name, System.getenv(name));
   }
+
+  // copied over from CLIUtils
+  public static void exit(int exitStatus) {
+    try {
+      System.exit(exitStatus);
+    } catch (java.lang.SecurityException secExc) {
+      if (exitStatus != 0)
+        throw new RuntimeException("SolrExporter failed to exit with status " + exitStatus);
+    }
+  }
+
+  // copied over from CLIUtils
+  private static String getDefaultSolrUrl() {
+    // note that ENV_VAR syntax (and the env vars too) are mapped to env.var sys props
+    String scheme = EnvUtils.getProperty("solr.url.scheme", "http");
+    String host = EnvUtils.getProperty("solr.tool.host", "localhost");
```

Comment:
We shouldn't use "solr.tool.host" here IMO, since that variable is going to go away, and I don't think it was ever used with the prometheus exporter.

Reply:
I just commited to 10x (only) the change to remove solr.tool.host.  backporting to 9x was tough.

Reply:
agreed we should just use `solr.host`

Reply:
makes sense. moved to `solr.host` variable